enterprise/ha/src/main/java/org/neo4j/kernel/ha/cluster/HighAvailabilityMemberState.java

```diff
@@ -59,11 +59,9 @@ public HighAvailabilityMemberState masterIsElected( HighAvailabilityMemberContex
                 public HighAvailabilityMemberState masterIsAvailable( HighAvailabilityMemberContext context,
                                                                       InstanceId masterId, URI masterHaURI )
                 {
-//                    assert context.getAvailableMaster() == null;
                     if ( masterId.equals( context.getMyId() ) )
                     {
-                        throw new RuntimeException( "Received a MasterIsAvailable event for my InstanceId while in" +
-                                " PENDING state" );
+                        return PENDING_ELECTION;
```

Comment:
Since `PENDING_ELECTION` will anyway delegate to `PENDING` everything, why not just use PENDING here as end state?